assets/js/contact_us_form_element.ts

```diff
@@ -39,6 +39,8 @@ class ContactUsFormElement extends HTMLElement {
     if (this.isInMaintenanceWindow) {
       this.showAlert();
       this.hide();
+    } else {
+      this.removeAlert();
     }
```

Comment:
this is not prose style updating, this is USWDS API updates righjt?

Reply:
This was a [workaround](https://github.com/GSA-TTS/identity-site/pull/1207#issuecomment-1936113605) to a [visual regression on the contact page](https://github.com/GSA-TTS/identity-site/pull/1207#issuecomment-1934972525) with the introduction of prose, since the use of sibling selectors in how prose creates margins doesn't account for hidden content. So rather than relying on it staying hidden, this removes it from the page to avoid the weirdness with the CSS sibling selectors.